charts/k8ssandra/values.yaml

```diff
@@ -80,6 +80,21 @@ cassandra:
    size:
    newGenSize:
 
+  gc:
```

Comment:
Why this sort of structure? Now we prevent use of ZGC or Shenandoah. I think I'd rather see a ConfigMap or such to make sure the user has to actually understand what they're doing (by having to define the parameters themselves).

Reply:
We cannot simply use a ConfigMap. We have to work within the constraints of the config-builder since that is what generate the `jvm-options` for 3.11 or the `jvm8-server.options` or `jvm11-server.options` for 4.0. We will need to update config-builder to support additional garbage collectors.